compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -7392,14 +7334,13 @@ fails, attempts to require sym's namespace and retries."
 
 (defn parse-boolean
   "Parse strings \"true\" or \"false\" and return a boolean, or nil if invalid"
-  [#_String s]
-  ;; (if (string? s)
-  ;;   (case s
-  ;;     "true" true
-  ;;     "false" false
-  ;;     nil)
-  ;;   (throw (IllegalArgumentException. (parsing-err s))))
-  (throw "TODO: port"))
+  [s]
+  (if (string? s)
+    (cond
```

Comment:
It feels reasonable to use `cond` here instead of `case`.

Reply:
Yep, let's get a TODO in there to use `case` though, since the perf is different.